lua/pounce.lua

```diff
@@ -54,9 +125,31 @@ local function calculate_proximity_bonus(cursor_line, cursor_col, match_line, ma
   return score
 end
 
-function M.setup(opts)
+M.config = function(opts)
   config = vim.tbl_extend("force", config, opts or {})
 end
+function M.setup(opts)
+  M.config(opts)
+
+  for hl, spec in pairs(default_hl) do
```

Comment:
I recently moved this to an autocmd so that colorschemes wouldn't reset it. Can the same be done in Lua? I also don't mind keeping the highlights in vimscript.